app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ConsumerCartFragment.kt

```diff
@@ -197,25 +236,54 @@ class ConsumerCartFragment : BaseFragment(R.layout.fragment_consumer_cart) {
                     oldTotalCost = "1650 ₽",
                     newTotalCost = "1500 ₽",
                 )
-            )
+            ),
+            onMenuClicked = {},
+            onErrorButtonClicked = { },
+            addProductToCartClicked = { s -> },
+            removeProductFromCartClicked = { s -> },
+            onProductClicked = { s -> },
+            onCreateOrderClicked = { },
         )
     }
 
     @Preview(showSystemUi = true)
     @Composable
     private fun ConsumerCartEmptyScreenPreview() {
-        ConsumerCartScreen(State.Empty())
+        ConsumerCartScreen(
```

Comment:
Do not forgot `FoodDeliveryTheme { ... }`

Reply:
dont forget